Tests/Acceptance/PaypalCheckoutTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Tests\Acceptance;
+
+use Wirecard\Oxid\Model\PaypalPaymentMethod;
+
+/**
+ * Acceptance tests for the PayPal checkout flow.
+ */
+class PaypalCheckoutTest extends CheckoutTestCase
+{
+    public function getPaymentMethod()
+    {
+        return new PaypalPaymentMethod();
+    }
+
+    public function testCheckoutForPurchase()
+    {
+        $this->setPaymentActionPurchase();
+        $this->goThroughCheckout();
```

Comment:
This method should be in the ChekoutTestCase as it will be used for each payment method, just pass the payment method as a parameter on what to click.

Reply:
There are payment methods (e.g. credit card), where within the checkout you already have to fill fields. See https://github.com/wirecard/oxid-ee/pull/84/files#diff-91b2dc1c3eb000c0a635e2e40467b225R78

The best I can do is create a base method in `ChekoutTestCase` and override it if necessary.

Reply:
Yes that's what i meant, for 80% of payment methods its the same.